utonics/labproject/main.go

```diff
@@ -319,13 +344,13 @@ func newProject(values map[string][]string, botClient, userClient *worker.Client
 	}
 
 	// Add Repositories to Team
-	msgs = append(msgs, fmt.Sprintf("Adding repository %q to team %q", project, team.Name))
-	if err := botClient.AdminAddTeamRepository(team.ID, project); err != nil {
+	msgs = append(msgs, fmt.Sprintf("Adding repositories %q to team %q", (project + ".main and others"), team.Name))
```

Comment:
This is already a `Sprintf()` with formatting directives.  Why append text to the `project` variable and not modify the formatting template (first argument)??